scheduler/scheduler.go

```diff
@@ -13,13 +16,23 @@ import (
 	"github.com/zalando/skipper/filters"
 	"github.com/zalando/skipper/metrics"
 	"github.com/zalando/skipper/routing"
+	"go.uber.org/atomic"
```

Comment:
Why not https://pkg.go.dev/sync/atomic ?

Reply:
Because we have to wait for Go 1.19 https://github.com/zalando/skipper/issues/2064